src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,34 @@
 package core.basesyntax;
 
 public class MainApp {
+    public MainApp(String[] machine) {
 
+        Machine truck = new Truck() {
+            @Override
+
+            public void doWork() {
+                super.doWork();
+            }
+        };
+        truck.doWork();
+        truck.stopWork();
+        Machine excavator = new Excavator() {
+            @Override
+
+            public void doWork() {
+                super.doWork();
+            }
+        };
+        excavator.doWork();
+        excavator.stopWork();
+        Machine bulldozer = new Bulldozer() {
+            @Override
+            public void doWork() {
+                super.doWork();
+            }
```

Comment:
The override of `doWork` for `Bulldozer` is redundant as well, as it doesn't add any new functionality. Consider instantiating `Bulldozer` without the anonymous class.